cip25-wasm/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cardano-multiplatform-lib-cip25",
+  "version": "3.1.1",
+  "description": "Multiplatform SDK for CIP25 (Cardano NFT Metadata)",
+  "scripts": {
+    "rust:build-nodejs": "rimraf ./pkg; wasm-pack build --target=nodejs; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:build-browser": "rimraf ./pkg; wasm-pack build --target=browser; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:build-web": "rimraf ./pkg; wasm-pack build --target=web; npm run js:ts-json-gen; wasm-pack pack",
+    "rust:publish": "cargo publish",
+    "rust:check-warnings": "(RUSTFLAGS=\"-D warnings\" cargo +stable build)",
+    "rust:test": "cargo test",
+    "js:prepublish": "npm run rust:test && rimraf ./publish && cp -r ./pkg ./publish && cp README.md publish/ && cp LICENSE publish/",
+"js:test-publish": "npm run rust:build-nodejs && npm run js:prepublish && node ../scripts/publish-helper cip25 cip25_wasm -nodejs && cd publish",
```

Comment:
We can remove this one. I just had it here to test out the publishing scripts since I had never used them and wasn't sure if running `npm publish` would do anything I didn't want to happen.